### PR TITLE
Add a basic scheduler download button

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR builds off of https://github.com/skyportal/skyportal/pull/2738 to add a button to download a possible schedule.

### CODE REVIEW

File: skyportal/handlers/api/followup_request.py [hunk 1]
```diff
@@ -596,3 +622,406 @@ def delete(self, request_id):
...
+def observation_schedule(
+    followup_requests,
+    instrument,
+    observation_start=Time.now(),
+    observation_end=Time.now() + TimeDelta(12 * u.hour),
+    output_format='csv',
+    figsize=(10, 8),
+):
+
+    """Create a movie to display observations of a given skymap
```
Comment (guynir42) [inline skyportal/handlers/api/followup_request.py:636]: Movie?

**Review by guynir42 — APPROVED** [review]
Just a minor documentation comment/question. Looks good overall. This will be very cool when we start using it.

File: skyportal/handlers/api/followup_request.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/handlers/api/followup_request.py]: If we were a movie.... you'd be the right guy!

**Comment by mcoughlin** [comment]
Minor documentation comment, which was addressed. Merging.

**Pull request merged**